modules/dnn/src/webgpu/include/tensor.hpp

```diff
@@ -12,7 +12,7 @@ class Buffer;
 class Tensor{
 public:
     Tensor(Format fmt = wFormatFp32);
-    Tensor( const void* data, std::vector<int>& shape,  
+    Tensor(const void* data, std::vector<int>& shape,  
             Format fmt = wFormatFp32);
```

Comment:
plase correct the wrong indents of paramters, not only for this function.

Reply:
> plase correct the wrong indents of paramters, not only for this function.

Thanks for your patience,  I've modified it. By the way, I've add the layer test for softmax in [this repo](https://github.com/NALLEIN/opencv/tree/layerTest),  please help to checkou whether this works on your device.

```
git clone https://github.com/NALLEIN/opencv.git
cd opencv/
git checkout -b layerTest origin/layerTest
mkdir build
cd build
cmake -D CMAKE_BUILD_TYPE=Release -DWITH_WEBGPU=ON -D CMAKE_INSTALL_PREFIX=/usr/local ..
make -j8
$(PATH_TO_OPENCV)/build/bin/opencv_test_dnn --gtest_filter=Layer_Test_Softmax.Accuracy
```
The test case is [here](https://github.com/NALLEIN/opencv/blob/layerTest/modules/dnn/test/test_layers.cpp#L1632). If there is nothing wrong with the test, I will start to complete other ops for evaluation 2.
@huningxin @wzw-intel  PTAL.

Reply:
test passed on my machine with nVidia GPU.